LocalSettings.php

```diff
@@ -2010,8 +2010,7 @@
 			'wikipedia',
 		],
 		'+mrjaroslavikwiki' => [
-			'wikipedia',
-			'cswikipedia',
+		        'wikipedia' => [ 'cs', 'en' ],
```

Comment:
could you please do it like you see below for zhdelwiki (ie different lines after the arrays ([])

Reply:
Better?